src/driver-logitech-g300.c

```diff
@@ -151,6 +151,54 @@ logitech_g300_raw_to_button_action(uint8_t data)
 	return NULL;
 }
 
+static uint8_t logitech_g300_modifier_to_raw(int modifier_flags)
+{
+	uint8_t modifiers = 0x00;
+
+	if (modifier_flags & MODIFIER_LEFTCTRL)
+		modifiers += 0x01;
```

Comment:
`|=` would be preferable if you are working on bit fields

Also do the same on the ones below